docs/guides/algorithmiq-tem.ipynb

```diff
@@ -213,7 +259,7 @@
     "-- | -- | -- | -- \n",
     "`max_bond_dimension` | int | The maximum bond dimension to be used for the tensor networks. | 500 | \n",
     "`tem_compression_cutoff` | float | The cutoff value to be used for the tensor networks. | 1e-16\n",
-    "`num_max_shots` | int | The maximum number of shots. | 100000\n",
+    "`max_execution_time` | int or `None` | The maximum execution time on the QPU in seconds. If the runtime exceeds this value, the job will be cancelled. If `None`, a default limit set by the IBM Runtime will apply. | `None`\n",
```

Comment:
Looks great, small modification: IBM Runtime --> Qiskit Runtime

Reply:
Corrected to Qiskit Runtime